layouts/css/base.styl

```diff
@@ -30,6 +30,8 @@ a:active
 a:hover
     color #fff
     background-color $node-green
+    code
```

Comment:
A little nit picking, but separating the ordinary `a:hover` styling from the `code` element style improves readability IMO.

ATM this results in a stylus parser error as you forgot to add `color` before the hex below :smile_cat: